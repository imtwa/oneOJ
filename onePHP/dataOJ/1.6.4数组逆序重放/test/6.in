19
2256 347 2096 176 1576 4396 3284 3160 2700 158 2653 1534 1116 3687 986 2681 1386 1458 1402 