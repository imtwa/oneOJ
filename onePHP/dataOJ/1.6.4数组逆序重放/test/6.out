1402 1458 1386 2681 986 3687 1116 1534 2653 158 2700 3160 3284 4396 1576 176 2096 347 2256 